opba-api-security/pom.xml

```diff
@@ -94,5 +94,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
```

Comment:
Can we avoid adding commons as a dependency?

Reply:
dependency removed